snet-ui/Schedule/ScheduleEvent.tsx

```diff
@@ -9,7 +9,7 @@ import { checkDateIsBetween, getDateInStandardFormat } from 'utils/date';
 import Typography from '@mui/material/Typography';
 import Grid from '@mui/material/Grid';
 import moment from 'moment';
-import { Container } from '@mui/material';
+import { Container, Divider } from '@mui/material';
```

Comment:
where is this `Divider` being used @dileep2503 ?

Reply:
Commited new changes and updated